code/controllers/subsystem/asset_loading.dm

```diff
@@ -7,7 +7,7 @@ SUBSYSTEM_DEF(asset_loading)
 	name = "Asset Loading"
 	priority = FIRE_PRIORITY_ASSET_LOADING
 	// todo: hibernation
-	subsystem_flags = SS_NO_INIT
+	subsystem_flags = SS_NO_INIT | SS_BACKGROUND
```

Comment:
Same applies here.

Reply:
asset loading is a lazy thing / not actually urgent (i just named it asest loading but it's more like 'lazyload stuff people haven't asked for yet', if someone asks it's loaded instantly)